src/immutable_data.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use crate::{XorName, XOR_NAME_LEN};
+use rust_sodium::crypto::sign::{PublicKey, PUBLICKEYBYTES};
+use tiny_keccak;
+
+#[derive(Hash, Eq, PartialEq, PartialOrd, Ord, Clone, Serialize, Deserialize)]
+pub struct UnpubImmutableData {
+    /// Contained ImmutableData.
+    data: Vec<u8>,
+    /// Contains a set of owners of this data. DataManagers enforce that a
+    /// DELETE or OWNED-GET type of request is coming from the
+    /// MaidManager Authority of the owners.
+    owners: PublicKey,
```

Comment:
This should be `threshold_crypto::PublicKeySet` (i.e. `BLS-PublicKey`).
If `threshold_crypto` is not included as a dependency, could you pls do this as well?

Reply:
Sure, I just saw it was commented out in Cargo.toml. I'll add it to the crate.